sources/packages/backend/apps/api/src/services/application-restriction-bypass/application-restriction-bypass.models.ts

```diff
@@ -0,0 +1,12 @@
+import { RestrictionBypassBehaviors } from "@sims/sims-db";
+
+export interface BypassRestrictionData {
+  applicationId: number;
+  studentRestrictionId: number;
+  bypassBehavior: RestrictionBypassBehaviors;
+  note: string;
+}
+
+export interface RemoveBypassRestrictionData {
```

Comment:
Just a clarification, a model does not need to be created for every service call.
The idea is just to allow services to be reused by other services without having the controllers DTOs as a parameter.
This one as a model is not a blocker but it would also be possible to be just a single parameter in the service method.